src/Backend/ServiceInterface.php

```diff
@@ -14,26 +14,26 @@ interface ServiceInterface
      *
      * @return RequestInterface
      */
-    public function getRequest($url);
+    public function getRequest($url): RequestInterface;
 
     /**
      * @param array $data
      *
      * @return int
      */
-    public function extractCount(array $data);
+    public function extractCount(array $data): int;
```

Comment:
If you require `int` here you need to add the `(int)` casts in most function of the implementing classes.

Reply:
Ah. It looks like it is better to rollback this change instead of casting return value inside every `extractCount` function

Reply:
Rolled back. Thank you for highlighting this